index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello from Gemini</title>
+</head>
+<body>
+    <header>
+    </header>
+    <main>
+    </main>
+    <footer>
+    </footer>
+</body>
+</html>
```

Comment:
This is a good start, but the `header`, `main`, and `footer` elements are all empty. Please add some content to these elements.